src/cli.rs

```diff
@@ -2,32 +2,42 @@
 
 use std::path::PathBuf;
 
-use clap::{AppSettings, ArgAction, ArgGroup, Parser};
+use clap::{
+    arg,
+    builder::{ArgAction, EnumValueParser},
+    command, ArgGroup, Parser,
+};
 
 use crate::types::{ColorOptions, PlatformType};
 
 // Note: flag names are specified explicitly in clap attributes
 // to improve readability and allow contributors to grep names like "clear-cache"
 #[derive(Parser, Debug)]
-#[clap(about = "A fast TLDR client", author, version)]
-#[clap(
-    after_help = "To view the user documentation, please visit https://dbrgn.github.io/tealdeer/."
+#[command(
+    about = "A fast TLDR client",
+    version = "1.2.3",
```

Comment:
Clap can read the version from `Cargo.toml`:
```suggestion
    version,
```